components/TxForm.tsx

```diff
@@ -1,34 +1,143 @@
 import useNotifications from "@/hooks/useNotifications";
+import useShieldPrivateKey from "@/hooks/useShieldPrivateKey";
```

Comment:
_(arbitrary comment location to start a thread)_

When I type into the token amount field I get this error

<img width="1536" alt="image" src="https://user-images.githubusercontent.com/17163988/212187891-2484f2d4-5a41-47c3-9ad9-aa2a101ce9a0.png">

Reply:
should be fixed now